site/docs/assets/scss/_markdown.scss

```diff
@@ -29,13 +25,26 @@
       padding-top: 0;
     }
   }
+  
+  h1.title {
+    font-size: 2.5em;
+    border: none !important;
+    font-family: 'Ubuntu', sans-serif !important;
+    font-weight: bold !important;
+  }
 
-  h3 {
+  h2 {
```

Comment:
Maybe h2 should set `font-family: 'Livvic', sans-serif;`

Reply:
h1 is previous h2. Current h1 has `'Livvic', sans-serif` style .

Reply:
Sorry, might be wrong here, this is just from observation not from css knowledge.
But previously h2 set the font-family, now is does not (same as h3) and they use the default from the body, which is Ubuntu. If that is the intention okay, but my assumption was that h? should use Livvic and only h1.title should use Ubuntu.

Reply:
Now I understand. Thanks for the explanation. If that's an assumption in the site design, yeah it should have that font. Let me revise. :+1:

Reply:
wait... Previous SCSS also doesn't have Livvic font in H2.

Reply:
Not sure if that is an assumption in the site design, but seemed like something to no change by mistake.
Well, h2 uses Livvic, but h3 does not.
Feels like we should have a clear understanding of how the site should look like, or rather if there are design rules which should be utilized.

Reply:
This PR doesn't use Livvic for H2, and H3 both. Only H1 uses Livvic font family.
Current master branch uses Livvic for H2.

Demoting headers means `#` to be visible as `<H2>`, `##` to be visible as `<H3>` in the older version. So, I've changed CSS that H1 to have old H2 style, H2 to have old H3 style..